lib/migrate.js

```diff
@@ -65,9 +68,17 @@ module.exports = function transformFile(currentConfigPath, outputConfigPath, opt
 				])
 				.then(answers => {
 					if (answers['confirmMigration']) {
-						// TODO validate the config
-						fs.writeFileSync(outputConfigPath, result, 'utf8');
-						console.log(chalk.green(`✔︎ New webpack v2 config file is at ${outputConfigPath}`));
+						fs.writeFile(outputConfigPath, result, 'utf8', (err) => {
+							const webpackOptionsValidationErrors = validateSchema(webpackOptionsSchema, require(outputConfigPath));
```

Comment:
Probably not needed. Fixing the transform issues and let's merge and take it from there.